packages/pages/src/dev/server/middleware/serverRenderSlugRoute.ts

```diff
@@ -3,7 +3,10 @@ import { ViteDevServer } from "vite";
 import { propsLoader } from "../ssr/propsLoader.js";
```

Comment:
unfortunate we need to duplicate the work here and in serverRenderRoute

Reply:
Good point. The separate files were introduced by Slapshot. I'll make a point to refactor this in a followup PR, but I won't change it in this one.